cfgov/jinja2/v1/_includes/molecules/related-metadata.html

```diff
@@ -135,13 +140,9 @@ <h3 class="h4">
    ========================================================================== #}
 
 {% macro _text(text) %}
-
-<p class="u-mb0">
     {{ parse_links(text.blob) | safe }}
-</p>
 {% endmacro %}
 
-{# TODO: This function is not used and can be removed. #}
```

Comment:
I just meant the `_topics()` function IS called, so the TODO comment was not accurate. However, it's currently not generating any markup when used in Wagtail, so I guess it's functionally not used.